applications/structural_application.orig/CMakeLists.txt

```diff
@@ -0,0 +1,139 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
```

Comment:
i have the feeling you are committing from a hidden directory (with the .orig in the name)

Reply:
thanks, I removed it in the latest commit